### ISSUE TITLE
Cannot find type definition file for 'core-js'.

### ISSUE THREAD

**Issue by hryktrd** [cesarrew/ng2-currency-mask#2]
I use this library in Angular 2 project.

but It output these error message.
```
[default] C:\Develop\ng2-primeng-sample\node_modules\ng2-currency-mask\src\currency-mask.directive.d.ts:1:0 
Cannot find type definition file for 'core-js'.
[default] C:\Develop\ng2-primeng-samle\node_modules\ng2-currency-mask\src\input.handler.d.ts:1:0 
Cannot find type definition file for 'core-js'.
```
And When I install core-js by npm i --save types@core-js, there are new errors as followings

```
ERROR in [default] C:\Develop\ng2-primeng-omine\node_modules\@types\core-js\index.d.ts:21:13
Duplicate identifier 'PropertyKey'.

ERROR in [default] C:\Develop\ng2-primeng-omine\node_modules\@types\core-js\index.d.ts:85:4
All declarations of 'name' must have identical modifiers.

ERROR in [default] C:\Develop\ng2-primeng-omine\node_modules\@types\core-js\index.d.ts:145:4
Subsequent variable declarations must have the same type.  Variable '[Symbol.unscopables]' must be of type '{ copyWithin: boolean;

ERROR in [default] C:\Develop\ng2-primeng-omine\node_modules\@types\core-js\index.d.ts:262:4
All declarations of 'flags' must have identical modifiers.

ERROR in [default] C:\Develop\ng2-primeng-omine\node_modules\@types\core-js\index.d.ts:276:4
All declarations of 'EPSILON' must have identical modifiers.

ERROR in [default] C:\Develop\ng2-primeng-omine\node_modules\@types\core-js\index.d.ts:311:4
All declarations of 'MAX_SAFE_INTEGER' must have identical modifiers.

ERROR in [default] C:\Develop\ng2-primeng-omine\node_modules\@types\core-js\index.d.ts:318:4
All declarations of 'MIN_SAFE_INTEGER' must have identical modifiers.

ERROR in [default] C:\Develop\ng2-primeng-omine\node_modules\@types\core-js\index.d.ts:457:4
Subsequent variable declarations must have the same type.  Variable '[Symbol.toStringTag]' must be of type '"Symbol"', but here has

ERROR in [default] C:\Develop\ng2-primeng-omine\node_modules\@types\core-js\index.d.ts:457:4
All declarations of '[Symbol.toStringTag]' must have identical modifiers.

ERROR in [default] C:\Develop\ng2-primeng-omine\node_modules\@types\core-js\index.d.ts:464:4
All declarations of 'prototype' must have identical modifiers.

ERROR in [default] C:\Develop\ng2-primeng-omine\node_modules\@types\core-js\index.d.ts:492:4
All declarations of 'hasInstance' must have identical modifiers.

ERROR in [default] C:\Develop\ng2-primeng-omine\node_modules\@types\core-js\index.d.ts:498:4
All declarations of 'isConcatSpreadable' must have identical modifiers.

ERROR in [default] C:\Develop\ng2-primeng-omine\node_modules\@types\core-js\index.d.ts:504:4
All declarations of 'iterator' must have identical modifiers.

ERROR in [default] C:\Develop\ng2-primeng-omine\node_modules\@types\core-js\index.d.ts:510:4
All declarations of 'match' must have identical modifiers.

ERROR in [default] C:\Develop\ng2-primeng-omine\node_modules\@types\core-js\index.d.ts:516:4
All declarations of 'replace' must have identical modifiers.

ERROR in [default] C:\Develop\ng2-primeng-omine\node_modules\@types\core-js\index.d.ts:522:4
All declarations of 'search' must have identical modifiers.

ERROR in [default] C:\Develop\ng2-primeng-omine\node_modules\@types\core-js\index.d.ts:528:4
All declarations of 'species' must have identical modifiers.

ERROR in [default] C:\Develop\ng2-primeng-omine\node_modules\@types\core-js\index.d.ts:534:4
All declarations of 'split' must have identical modifiers.

ERROR in [default] C:\Develop\ng2-primeng-omine\node_modules\@types\core-js\index.d.ts:540:4
All declarations of 'toPrimitive' must have identical modifiers.

ERROR in [default] C:\Develop\ng2-primeng-omine\node_modules\@types\core-js\index.d.ts:546:4
All declarations of 'toStringTag' must have identical modifiers.

ERROR in [default] C:\Develop\ng2-primeng-omine\node_modules\@types\core-js\index.d.ts:552:4
All declarations of 'unscopables' must have identical modifiers.

ERROR in [default] C:\Develop\ng2-primeng-omine\node_modules\@types\core-js\index.d.ts:609:4
Subsequent variable declarations must have the same type.  Variable '[Symbol.toStringTag]' must be of type '"Math"', but here has t

ERROR in [default] C:\Develop\ng2-primeng-omine\node_modules\@types\core-js\index.d.ts:609:4
All declarations of '[Symbol.toStringTag]' must have identical modifiers.

ERROR in [default] C:\Develop\ng2-primeng-omine\node_modules\@types\core-js\index.d.ts:613:4
Subsequent variable declarations must have the same type.  Variable '[Symbol.toStringTag]' must be of type '"JSON"', but here has t

ERROR in [default] C:\Develop\ng2-primeng-omine\node_modules\@types\core-js\index.d.ts:613:4
All declarations of '[Symbol.toStringTag]' must have identical modifiers.

ERROR in [default] C:\Develop\ng2-primeng-omine\node_modules\@types\core-js\index.d.ts:628:4
All declarations of 'size' must have identical modifiers.

ERROR in [default] C:\Develop\ng2-primeng-omine\node_modules\@types\core-js\index.d.ts:634:4
All declarations of 'prototype' must have identical modifiers.

ERROR in [default] C:\Develop\ng2-primeng-omine\node_modules\@types\core-js\index.d.ts:645:4
All declarations of 'size' must have identical modifiers.

ERROR in [default] C:\Develop\ng2-primeng-omine\node_modules\@types\core-js\index.d.ts:651:4
All declarations of 'prototype' must have identical modifiers.

ERROR in [default] C:\Develop\ng2-primeng-omine\node_modules\@types\core-js\index.d.ts:666:4
All declarations of 'prototype' must have identical modifiers.

ERROR in [default] C:\Develop\ng2-primeng-omine\node_modules\@types\core-js\index.d.ts:680:4
All declarations of 'prototype' must have identical modifiers.

ERROR in [default] C:\Develop\ng2-primeng-omine\node_modules\@types\core-js\index.d.ts:692:4
All declarations of 'value' must have identical modifiers.

ERROR in [default] C:\Develop\ng2-primeng-omine\node_modules\@types\core-js\index.d.ts:804:4
All declarations of 'prototype' must have identical modifiers.

ERROR in [default] C:\Develop\ng2-primeng-omine\node_modules\typescript\lib\lib.es2015.core.d.ts:17:13
```

so I tried to delete `/// <reference types="core-js" />` from currency-mask.directive.d.ts and input.handler.d.ts.
After this solution, Error isn't appeared, but I think that this solution is not good and will be cause of other error.
but there is no error now.
Is this a correct solution ?

I use angular 2.1.1, PrimeNG 1.0.0 and current of ng2-currency-mask.

Thank you.

**Comment by cesarrew** [comment]
I guess that the problem was fixed in version 1.0.6. Could you test it?

**Comment by hryktrd** [comment]
Thank you.
I tested and I see that problem has fixed.
